src/layouts/components/Editor/components/MenuBar.tsx

```diff
@@ -275,32 +275,62 @@ export const MenuBar = ({ editor }: { editor: Editor }) => {
     },
     {
       type: "divider",
-      isHidden: !useIsIsomerFeatureOn(FEATURE_FLAGS.IS_COMPLEX_BLOCKS_ENABLED),
+      isHidden:
+        useFeatureValue<{ blocks: string[] }>(
+          FEATURE_FLAGS.RTE_ENABLED_BLOCKS,
+          {
+            blocks: [],
+          }
+        ).blocks.length === 0,
     },
     {
       type: "detailed-list",
       label: "Add complex blocks",
       icon: BiPlus,
-      isHidden: !useIsIsomerFeatureOn(FEATURE_FLAGS.IS_COMPLEX_BLOCKS_ENABLED),
+      isHidden:
+        useFeatureValue<{ blocks: string[] }>(
+          FEATURE_FLAGS.RTE_ENABLED_BLOCKS,
+          {
+            blocks: [],
+          }
+        ).blocks.length === 0,
```

Comment:
i think this call of 
```ts
 useFeatureValue<{ blocks: string[] }>(
          FEATURE_FLAGS.RTE_ENABLED_BLOCKS,
          {
            blocks: [],
          }
        )
```
is repeated quite often throughout this file and might benefit from being extracted into a variable, for example, `rteEnabledBlocks` for readability purposes

Reply:
@seaerchin thanks, good catch!
Refactored in b28f0e0daf9e2b01cde1c9295b161fb9bc2e3908

Created 2 reusable variables: 
```
const rteEnabledBlocks = useFeatureValue<{ blocks: RTEBlockValues[] }>(
    FEATURE_FLAGS.RTE_ENABLED_BLOCKS,
    {
      blocks: [],
    }
  ).blocks

  const isRteEnabledBlocksEmpty = rteEnabledBlocks.length === 0
```